utils/enumeration.py

```diff
@@ -0,0 +1,56 @@
+class Enumeration:
```

Comment:
This can easily enum values and check if ti contains a value or not. Enum uses values as hash of name+value, advanced behaviour that is not needed for this extremely simple case and is annoying.

Reply:
`Enum` can really be used for this purpose.
~~~python
import enum

class Model(enum.Enum):
    COCO = enum.auto()
    BODY_25 = enum.auto()

for model in Model:  # like Enumeration.enum_values
    print(model)

for name, member in Model.__members__.items(): # like Enumeration.enum_values
    print('name={}, member={}'.format(name,member))

if Model.COCO in Model:  # like Enumeration.contains
    print('Model.COCO is in Model')
~~~

Reply:
381f6a86a7ebec3a1ee4f9edcac78c06ae77d08f